sync/src/tests/helpers.rs

```diff
@@ -420,8 +455,8 @@ impl TestNet {
 		self.peers.iter().all(|p| p.queue.is_empty())
 	}
 
-	pub fn trigger_block_verified(&mut self, peer_id: usize) {
+	pub fn trigger_chain_new_blocks(&mut self, peer_id: usize) {
```

Comment:
If I understand correctly it doesn't necessarily mean that blocks went to canon chain. I'm open to any different name, though.